api/lib/ApiService.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var config = require('./config.js').config;
+var pg = require('pg');
+var error = require('./config.js').errors;
+
+function checkArguments (args) {
```

Comment:
This check seems useless. Can you add some comments and/or tests to prove me wrong ? :)